topics/single-cell/tutorials/alevin-commandline/tutorial.md

```diff
@@ -117,6 +122,13 @@ To generate gene-level quantifications based on transcriptome quantification, Al
 
 We will use the murine reference annotation as retrieved from Ensembl (*GRCm38* or *mm10*) in GTF format. This annotation contains gene, exon, transcript and all sorts of other information on the sequences. We will use these to generate the transcript-gene mapping by passing that information to a tool that extracts just the transcript identifiers we need.
 
+Finally, we will download some files for later so that you don't have to switch kernels again - that's an Alevin output folder for another sample!
```

Comment:
Why do you need to change the kernel that frequently?

Reply:
Indeed that wasn't needed, so I just moved bit of code from the later part of the tutorial to the beginning so that everything is done altogether. Currently it's just bash (Salmon & Alevin) -> R (tximeta, Dropletutils, SCE)